modules/ocf/manifests/packages/postfix.pp

```diff
@@ -1,39 +1,36 @@
-class ocf::packages::postfix {
+class ocf::packages::postfix($manage = true) {
```

Comment:
I think it would be clearer if these kinds of variables had their default values defined in `common.yaml`, and then were loaded with a `hiera()` call to make it more explicit where the value is coming from. Otherwise I can see someone (or me) searching all the manifests trying to find where this is passed as an argument and being quite confused until they realize that hiera is overwriting this value. Or worse, sometimes this might be passed as an argument in some places and in other places as a hiera value, which would be even harder to find.